src/main/java/me/jellysquid/mods/sodium/client/gl/tessellation/TessellationBinding.java

```diff
@@ -4,10 +4,13 @@
 import me.jellysquid.mods.sodium.client.gl.buffer.GlBuffer;
 import me.jellysquid.mods.sodium.client.gl.buffer.GlBufferTarget;
 
+import java.util.Objects;
+
 public record TessellationBinding(GlBufferTarget target,
                                   GlBuffer buffer,
                                   GlVertexAttributeBinding[] attributeBindings) {
     public static TessellationBinding forVertexBuffer(GlBuffer buffer, GlVertexAttributeBinding[] attributes) {
+        Objects.requireNonNull(attributes);
```

Comment:
why does this exist?

Reply:
To ensure the `null` return value from `DefaultChunkRenderer#getBindingsForType` is caught if for some reason a mixin does not replace it. Returning null there is a dirty hack to ensure binary compatibility with one of the Iris mixins; we could restructure all of this if we're fine breaking addons (I prefer not to, hence the slightly weird choices).

Reply:
maybe this possible nullability could be documented